src/debt_order/index.ts

```diff
@@ -1,19 +1,47 @@
 import * as _ from "lodash";
+import { BigNumber } from "../../utils/bignumber";
+
 import { Dharma } from "../";
+import { CollateralizedSimpleInterestLoanOrder } from "../adapters/collateralized_simple_interest_loan_adapter";
 import { DebtOrderData, ECDSASignature, InterestRate, Term, TokenAmount } from "../types";
 
+export interface DebtOrderParams {
+    principal: TokenAmount;
+    collateral: TokenAmount;
+    interestRate: InterestRate;
+    term: Term;
+    debtor: string;
```

Comment:
Can we standardize this to either `debtor` or `debtorAddress`?  For the creditor side, I've used `creditorAddress`.  I feel the latter is more specific but am not married to either.